### PR TITLE
Setting up LESS structure and main files for styling

### PR DESCRIPTION
**Pull request by MikeLikesCode**
What I did:

- Made LESS files for styling. Also added folders for future use in styling components.

What files changed:

Created:

- InstructorStyles folder
- ParentStyles folder
- global.less
- index.less
- reset.less

Why it was changed:

- Adding a structure to be used for when adding styling to components and pages.

[Loom video] !(https://www.loom.com/share/dc22311299314c0d80a74517de2e43d2)

### CODE REVIEW

**Review by cn8817 — APPROVED** [review]
Looks good to me!

File: src/styles/reset.less [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/**
```
Comment (bradlylewis) [inline src/styles/reset.less:1]: Everything looks good.

**Review by bradlylewis — APPROVED** [review]

**Review by fernando817mm — APPROVED** [review]
Awesome, thank you for setting this up!

**Review by Jie-chelchel — APPROVED** [review]
Looks good!

**Pull request merged**